united_survey/urls.py

```diff
@@ -8,5 +8,6 @@
     path("admin/", admin.site.urls),
     path("", include("organization.urls")),
```

Comment:
in router URLS its better to route path to apps. Instead of empty "", path to sruvey app "survey"